src/content/fr/2020/performance.md

```diff
@@ -0,0 +1,388 @@
+  description="Distribution de sites web ayant des performances rapides, moyennes et lentes sur le bureau. La distribution des sites web rapides semble linéaire, avec un moindre renflement que celui observé sur le graphique des ordinateurs de bureau. Le nombre d’expériences rapides et lentes du FCP est plus élevé qu’en 2019, tandis que le nombre d’expériences moyennes diminue en raison des changements de catégorisation du FCP.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=38297781&format=interactive",
+  sheets_gid="2122167666",
+  sql_file="web_vitals_distribution_by_device.sql"
+  )
+}}
+
+Dans les graphiques ci-dessus, les distributions de FCP sont réparties entre les ordinateurs de bureau et les téléphones portables. [Par rapport à l’année dernière](../2019/performance#fcp-by-device), on constate une diminution notable des relevés FCP moyens, tandis que le pourcentage d’expériences d’utilisation rapides et lentes a augmenté quel que soit le type d’appareil. Nous pouvons toujours observer la même tendance, à savoir que les personnes utilisant des téléphones portables connaissent plus fréquemment des FCP plus lents que celles qui utilisent des ordinateurs de bureau. Dans l’ensemble, ces personnes sont plus susceptibles d’avoir une bonne ou une mauvaise expérience, plutôt qu’une expérience moyenne.
+
+{{ figure_markup(
+  image="performance-fcp-mobile-year-on-year.png",
+  caption="Une comparaison de la distribution des sites web ayant un bon FCP, un FCP nécessitant d’être amélioré et un FCP ayant une mauvaise performance mobile entre 2019 et 2020.",
+  description="Diagramme à barres montrant la manière dont ont évolué la répartition des bons FCP, des FCP à améliorer et des mauvais FCP sur mobiles entre 2019 et 2020. En 2020, 75 % des sites web ont un FCP de moindre qualité.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=2037503700&format=interactive",
+  sheets_gid="1270303192",
+  sql_file="web_vitals_by_device.sql"
+  )
+}}
+
+En comparant le FCP sur les appareils mobiles sur une base annuelle, nous observons moins de bonnes expériences et plus d’expériences modérées et mauvaises. 75 % des sites web ont un FCP inférieur à la moyenne. Nous pouvons supposer que ce pourcentage élevé de mesures de FCP inférieures à l’idéal est une source de frustration et de dégradation de l’expérience d’utilisation.
+
+De nombreux facteurs peuvent retarder le rendu, comme la latence du serveur (mesurée par une poignée d’indicateurs, tels que [<span lang="en">Time to First Byte</span> (TTFB)](#time-to-first-byte-ttfb) et RTT), le blocage des requêtes JavaScript, ou la manipulation inappropriée des polices personnalisées, pour n’en citer que quelques-uns.
```

Comment:
```suggestion
De nombreux facteurs peuvent retarder le rendu, comme la latence du serveur (mesurée par une poignée d’indicateurs, tels que [<span lang="en">Time to First Byte</span> (TTFB)](#time-to-first-byte) et RTT), le blocage des requêtes JavaScript, ou la manipulation inappropriée des polices personnalisées, pour n’en citer que quelques-uns.
```

Any chance you could fix this in English version too as part of this PR?

Reply:
Done 👍